Source/Urho3D/Graphics/CustomGeometry.cpp

```diff
@@ -204,6 +204,34 @@ bool CustomGeometry::DrawOcclusion(OcclusionBuffer* buffer)
     return success;
 }
 
+void CustomGeometry::MakeCircle(float radius, int inc, Vector3 normal, float startTheta, float endTheta)

```

Comment:
can you name `inc` parameter differently to something like `iterations` maybe? Just so it better explains what it does when reading only header files.